rwgame/RWGame.cpp

```diff
@@ -75,6 +74,10 @@ RWGame::RWGame(Logger& log, int argc, char* argv[])
     renderer.text.setFontTexture(FONT_PRICEDOWN, "font1");
     renderer.text.setFontTexture(FONT_ARIAL, "font2");
 
+    hudDrawer = HUDDrawer();
```

Comment:
You can avoid this line, if you convert 35th line of `RWGame.hpp` to `HUDDrawer hudDrawer{};`.